### PR TITLE
Added Avado as solo staking tool

### PR DESCRIPTION
**Pull request by heeckhau**
Added Avado as solo staking solution

## Description

Avado is the easiest hardware solution for home staking

## Related Issue

https://github.com/ethereum/ethereum-org-website/issues/6340

### CODE REVIEW

**Comment by heeckhau** [comment]
@pettinarip Anything I can do to make progress here? I don't have a Vercel account

**Comment by minimalsm** [comment]
Hey @heeckhau, thanks for the PR. You don't need to worry about the Vercel message. We'll try to review this very soon.

**Comment by corwintines** [comment]
@wackerow do you have a moment to review this?

File: src/data/staking-products.json [hunk 1]
```diff
@@ -190,6 +190,33 @@
         "eventAction": "Clicked",
         "eventName": "Clicked Vouch + Dirk go to link"
       }
+    },
+    {
+      "name": "AVADO",
```
Comment (wackerow) [inline src/data/staking-products.json:195]: Curious if there would be opposition to making this title case consistent with the others?
```suggestion
      "name": "Avado",
```

File: src/data/staking-products.json [hunk 2]
```diff
@@ -190,6 +190,33 @@
...
+      "svgPath": "avado-glyph.svg",
+      "hue": 239,
+      "launchDate": "2018-12-01",
+      "url": "https://ava.do/",
+      "audits": [],
+      "minEth": 16,
```
Comment (wackerow) [inline src/data/staking-products.json:201]: Good to see support for Rocket Pool... 
I know this isn't fully utilized on the front-end yet, but similar to the Rocket Pool entry we should include the additional deposit requirements in the JSON here.
```suggestion
      "minEth": 16,
      "additionalStake": "10",
      "additionalStakeUnit": "% RPL",
```

File: src/data/staking-products.json [hunk 3]
```diff
@@ -190,6 +190,33 @@
...
+      "isFoss": true,
+      "hasBugBounty": false,
+      "isTrustless": true,
+      "isPermissionless": true,
+      "multiClient": true,
+      "easyClientSwitching": true,
+      "platforms": ["Linux", "macOS", "Windows", "Browser"],
```
Comment (wackerow) [inline src/data/staking-products.json:208]: My understanding is the Avado interface is accessed through a browser, and not as stand-along executable (similar to DAppNode above). Am I incorrect?
```suggestion
      "platforms": ["Browser"],
```

**Review by wackerow — CHANGES_REQUESTED** [review]
@heeckhau Hey, left a couple questions and suggestions, let me know your thoughts and I think we can get this in!

**Comment by heeckhau** [comment]
@wackerow Thanks a lot for the suggestions. I have applied the improvements.

File: src/data/staking-products.json [hunk 4]
```diff
@@ -250,6 +279,32 @@
         "eventAction": "Clicked",
         "eventName": "Clicked ethdo go to link"
       }
+    },
+    {
+      "name": "AVADO",
```
Comment (corwintines) [inline src/data/staking-products.json:284]:
```suggestion
      "name": "Avado",
```
Just changing this to be consistent

File: src/data/staking-products.json [hunk 5]
```diff
@@ -250,6 +279,32 @@
...
+      "launchDate": "2018-12-01",
+      "url": "https://ava.do/",
+      "audits": [],
+      "minEth": 16,
+      "isFoss": true,
+      "hasBugBounty": false,
+      "isTrustless": true,
+      "isPermissionless": true,
+      "isSelfCustody": true,
+      "platforms": ["Linux", "macOS", "Windows", "Browser"],
```
Comment (corwintines) [inline src/data/staking-products.json:296]:
```suggestion
      "platforms": ["Browser"],
```

Changing this for consistency as well

File: src/data/staking-products.json [hunk 6]
```diff
@@ -250,6 +279,32 @@
...
       }
+    },
+    {
+      "name": "AVADO",
+      "svgPath": "avado-glyph.svg",
+      "hue": 239,
+      "launchDate": "2018-12-01",
+      "url": "https://ava.do/",
+      "audits": [],
+      "minEth": 16,
```
Comment (corwintines) [inline src/data/staking-products.json:290]:
```suggestion
      "minEth": 16,
      "additionalStake": "10",
      "additionalStakeUnit": "% RPL",
```

Here as well

**Review by corwintines — APPROVED** [review]
Thanks, @heeckhau!

**Pull request merged**